torch/csrc/jit/script/module.cpp

```diff
@@ -33,15 +33,15 @@ static ModulePtr create_module_object(
 }
 
 Module::Module(c10::QualifiedName class_name)
-    : module_value_(create_module_object(
+    : Module(create_module_object(
```

Comment:
This is a weird construction because it sort of looks like `Module` inherits from `Module`. Can we just keep it the way it was?

Reply:
These changes were so that all the modules constructors would delegate to a single one that could register training, but that's not necessary in the latest commit